clang/lib/Format/WhitespaceManager.cpp

```diff
@@ -111,6 +111,9 @@ const tooling::Replacements &WhitespaceManager::generateReplacements() {
   alignConsecutiveDeclarations();
   alignConsecutiveBitFields();
   alignConsecutiveAssignments();
+  alignConsecutiveTableGenCondOperatorColons();
+  AlignConsecutiveTableGenBreakingDAGArgColons();
+  alignConsecutiveTableGenDefinition();
```

Comment:
Shouldn't these replacements be guarded behind an `isTableGen`? At the very least, `alignConsecutiveTableGenDefinition` aligns on TT_InheritanceColon and that can appear in non-tablegen code too